<!DOCTYPE HTML>
<html lang="en">
    <head>
        <title>Brandon Carter</title>
        <meta charset="utf-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <meta name="description" content="About Brandon Carter">
        <meta name="keywords" content="Brandon,Carter,MIT,ThinkTx,researcher,software,machine,learning,biology">
        <meta name="author" content="Brandon Carter">
        <link rel="stylesheet" 
href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css">
        <link rel="preconnect" href="https://fonts.googleapis.com">
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,400&display=swap" rel="stylesheet">
        <link rel="stylesheet" href="style.css">
        <!-- Google tag (gtag.js) -->
        <script async src="https://www.googletagmanager.com/gtag/js?id=G-J14EN89JM1"></script>
        <script>
          window.dataLayer = window.dataLayer || [];
          function gtag(){dataLayer.push(arguments);}
          gtag('js', new Date());

          gtag('config', 'G-J14EN89JM1');
        </script>
    </head>
    <body id="top">
        <!-- Main -->
        <div id="main">
            <!-- Profile -->
            <div id="profile">
                <div class="image avatar"><img src="images/avatar.jpg" alt="Brandon Profile Picture" /></div>
                <h1>Brandon Carter</h1>
                <span class="header-subtext">CTO @ ThinkTx</span>
                <div class="social-icons">
                    <a href="https://www.linkedin.com/in/b-carter" target="_blank" class="fab fa-linkedin" aria-label="LinkedIn"></a>
                    <a href="https://github.com/b-carter" target="_blank" class="fab fa-github" aria-label="GitHub"></a>
                    <a href="https://scholar.google.com/citations?user=MwzQlyIAAAAJ&hl=en" target="_blank" class="fab fa-google-scholar" aria-label="GitHub"></a>
                </div>
                <div id="email"><a href="mailto:bcarter@csail.mit.edu">bcarter@csail.mit.edu</a></div>
            </div>

            <!-- About Me -->
            <div id="aboutme-text">
                <p>I am the CTO at <a href="https://thinktx.co">Think Therapeutics</a>. Previously, I completed my Ph.D. in Computer Science at <a href="https://www.csail.mit.edu">MIT CSAIL</a> advised by <a href="https://cgs.csail.mit.edu">Prof. David Gifford</a>. I developed methods to interpret deep neural networks and design therapeutics using machine learning.</p>
                <p>Prior to that, I was an undergrad (MIT '17) and MEng student at MIT. I double majored in computer science
                    and mathematics and minored in economics.</p>
                <p>I have had the pleasure to work at Google Brain, Facebook, Bloomberg LP, KAYAK, and Leiden University.</p>
                <p>I am originally from Long Island, New York. In my free time I enjoy skiing and sailing.</p>
            </div>
        </div>

        <!-- Publications -->
        <section id="publications">
            <h2>Publications</h2>
            <p><b><a target="_blank" href="https://www.frontiersin.org/articles/10.3389/fimmu.2023.1135815/full">A pan-variant mRNA-LNP T cell vaccine protects HLA transgenic mice from mortality after infection with SARS-CoV-2 Beta</a></b><br>
                            Brandon Carter*, Pinghan Huang*, Ge Liu, Yuejin Liang, Paulo J.C. Lin, Bi-Hung Peng, Lindsay G. A. McKay, Alexander Dimitrakakis, Jason Hsu, Vivian Tat, Panatda Saenkham-Huntsinger, Jinjin Chen, Clarety Kaseke, Gaurav D. Gaiha, Qiaobing Xu, Anthony Griffiths, Ying K. Tam, Chien-Te K. Tseng, David K. Gifford<br>
                            <i>Frontiers in Immunology, 2023</i><br>
                            <span class="smalltext">[<a target="_blank" href="https://news.mit.edu/2023/creating-versatile-vaccine-take-on-covid-19-0309">Press &#8211; MIT News</a>]</span> <span class="smalltext">[<a target="_blank" href="https://www.bostonglobe.com/2023/03/09/business/local-scientists-hope-create-ultimate-covid-vaccine/">Press &#8211; Boston Globe</a>]</span>
            </p>
            <p><b><a target="_blank" href="https://arxiv.org/abs/2101.10902">Maximum n-times Coverage for Vaccine Design</a></b><br>
                Ge Liu, Alexander Dimitrakakis, Brandon Carter, David Gifford<br>
                <i>International Conference on Learning Representations (ICLR), 2022</i><br>
                <span class="smalltext">[<a href="https://github.com/gifford-lab/optivax">Code</a>]</span>
            </p>
            <p><b><a target="_blank" href="https://dl.acm.org/doi/10.1145/3490099.3511122">Embedding Comparator: Visualizing Differences in Global Structure and Local Neighborhoods via Small Multiples</a></b><br>
                Angie Boggust*, Brandon Carter*, Arvind Satyanarayan<br>
                <i>International Conference on Intelligent User Interfaces (IUI), 2022</i><br>
                <span class="smalltext">[<a href="http://vis.mit.edu/embedding-comparator/">Demo</a>]</span> <span class="smalltext">[<a href="https://www.youtube.com/watch?v=UU5LAxF8-7Q">Video</a>] [<a href="https://github.com/mitvis/embedding-comparator">Code</a>]</span>
            </p>
            <p><b><a target="_blank" href="https://www.nature.com/articles/s41587-021-01179-w">Using Deep Learning to Classify the Protein Universe</a></b><br>
                Maxwell Bileschi, David Belanger, Drew Bryant, Theo Sanderson, Brandon Carter, D. Sculley, Alex Bateman, Mark DePristo, Lucy Colwell<br>
                <i>Nature Biotechnology, 2022</i><br>
                <span class="smalltext">[<a href="https://www.bloomberg.com/news/features/2019-07-15/google-ai-could-challenge-big-pharma-in-drug-discovery">Press</a>]</span>
            </p>
            <p><b><a target="_blank" href="https://arxiv.org/abs/2003.08907">Overinterpretation reveals image classification model pathologies</a></b><br>
                Brandon Carter, Siddhartha Jain, Jonas Mueller, David Gifford<br>
                <i>Advances in Neural Information Processing Systems (NeurIPS), 2021</i><br>
                <span class="smalltext">[<a target="_blank" href="https://news.mit.edu/2021/nonsense-can-make-sense-machine-learning-models-1215">Press</a>]</span>
                <span class="smalltext">[<a href="https://github.com/gifford-lab/overinterpretation">Code</a>]</span>
            </p>
            <p><b><a target="_blank" href="https://www.cell.com/cell-systems/fulltext/S2405-4712(20)30461-0">Predicted Cellular Immunity Population Coverage Gaps for SARS-CoV-2 Subunit Vaccines and their Augmentation by Compact Peptide Sets</a></b><br>
                Ge Liu, Brandon Carter, David Gifford<br>
                <i>Cell Systems, 2021</i><br>
                <span class="smalltext">[<a target="_blank" href="https://www.csail.mit.edu/news/mit-study-covid-19-vaccines-may-be-less-effective-asian-americans">Press</a>]</span>
                <span class="smalltext">[<a href="https://github.com/gifford-lab/optivax/tree/master/augmentation_paper">Code</a>]</span>
            </p>
            <p><b><a target="_blank" href="https://doi.org/10.1093/bioinformatics/btab131">Machine learning optimization of peptides for presentation by class II MHCs</a></b><br>
                Zheng Dai*, Brooke Huisman*, Haoyang Zeng, Brandon Carter, Siddhartha Jain, Michael Birnbaum, David Gifford<br>
                <i>Bioinformatics, 2021</i><br>
                <span class="smalltext">[Featured as spotlight talk at MLCB 2019]</span>
            </p>
            <p><b><a target="_blank" href="https://arxiv.org/abs/2103.03014">Lost in Pruning: The Effects of Pruning Neural Networks beyond Test Accuracy</a></b><br>
                Lucas Liebenwein, Cenk Baykal, Brandon Carter, David Gifford, Daniela Rus<br>
                <i>Machine Learning and Systems (MLSys), 2021</i><br>
                <span class="smalltext">[<a href="https://github.com/lucaslie/torchprune">Code</a>]</span>
            </p>
            <p><b><a target="_blank" href="https://www.cell.com/cell-systems/fulltext/S2405-4712(20)30238-6">Computationally Optimized SARS-CoV-2 MHC Class I and II Vaccine Formulations Predicted to Target Human Haplotype Distributions</a></b><br>
                Ge Liu*, Brandon Carter*, Trenton Bricken, Siddhartha Jain, Mathias Viard, Mary Carrington, David Gifford<br>
                <i>Cell Systems, 2020</i><br>
                <span class="smalltext">[<a target="_blank" href="https://www.csail.mit.edu/news/potential-covid-19-vaccines-get-boost-machine-learning">Press</a>]</span>
                <span class="smalltext">[<a href="https://github.com/gifford-lab/optivax">Code</a>]</span>
            </p>
            <p><b><a target="_blank" href="https://academic.oup.com/bioinformatics/article/36/7/2126/5645171">Antibody complementarity determining region design using high-capacity machine learning</a></b><br>
                Ge Liu*, Haoyang Zeng*, Jonas Mueller, Brandon Carter, Ziheng Wang, Jonas Schilz, Geraldine Horny, Michael Birnbaum, Stefan Ewert, David Gifford<br>
                <i>Bioinformatics, 2020</i><br>
                <span class="smalltext">[<a href="https://github.com/gifford-lab/antibody-2019">Code</a>]</span>
            </p>
            <p><b><a target="_blank" href="https://arxiv.org/abs/1810.03805">What made you do this? Understanding black-box decisions with sufficient input subsets</a></b><br>
                Brandon Carter*, Jonas Mueller*, Siddhartha Jain, David Gifford<br>
                <i>Artificial Intelligence and Statistics (AISTATS), 2019</i><br>
                <span class="smalltext">[Featured as contributed talk at <a href="https://irasl.gitlab.io/">NeurIPS 2018 Workshop on Interpretability and Robustness</a>] [<a href="https://people.csail.mit.edu/bcarter/sis_neurips_irasl_slides.pdf">Slides</a>] [<a href="https://mit6874.github.io/assets/slides/6.874-lecture-5-2019.pdf">Lecture notes</a>] [<a href="https://github.com/google-research/google-research/tree/master/sufficient_input_subsets">Code</a>]</span>
            </p>
            <p><b><a target="_blank" href="https://www.biorxiv.org/content/10.1101/674119v1">Critiquing Protein Family Classification Models Using Sufficient Input Subsets</a></b><br>
                Brandon Carter, Maxwell Bileschi, Jamie Smith, Theo Sanderson, Drew Bryant, David Belanger, Lucy Colwell<br>
                <i>Journal of Computational Biology, 2019</i><br>
                <span class="smalltext">[Featured as spotlight talk at <a href="https://sites.google.com/view/icml-compbio-2019">ICML 2019 Workshop on Computational Biology</a>] [<a href="https://people.csail.mit.edu/bcarter/icml_wcb_2019_slides.pdf">Slides</a>]</span>
            </p>
            <p><b><a target="_blank" href="https://courses.csail.mit.edu/6.857/2016/files/2.pdf">Survey of Fully Verifiable Voting Cryptoschemes</a></b><br>
                Brandon Carter, Kenneth Leidal, Devin Neal, Zachary Neely<br>
                <i>MIT Computer and Network Security (6.857) Final Project, 2016</i><br>
            </p>
            <p><b><a target="_blank" href="http://iovs.arvojournals.org/article.aspx?articleid=2359902">Safety and Efficacy of Ganciclovir Ophthalmic Gel for Treatment of Adenovirus Keratoconjunctivitis Utilizing Cell Culture and Animal Models</a></b><br>
                Seth Epstein, Karen Fernandez, Brandon Carter, Salma Abdou, Neha Gadaria, Penny Asbell<br>
                <i>Investigative Ophthalmology and Visual Science (IOVS), 2012</i><br>
            </p>
            <p><b><a target="_blank" href="https://dspace.mit.edu/handle/1721.1/151487">Interpretations of Machine Learning and Their Application to Therapeutic Design</a></b><br>
                Brandon Carter<br>
                <i>Ph.D. Thesis, MIT Dept. of Electrical Engineering and Computer Science, 2023</i><br>
            </p>
            <p><b><a target="_blank" href="https://dspace.mit.edu/bitstream/handle/1721.1/123008/1127567462-MIT.pdf?sequence=1&isAllowed=y">Interpreting Black-Box Models Through Sufficient Input Subsets</a></b><br>
                Brandon Carter<br>
                <i>M.Eng. Thesis, MIT Dept. of Electrical Engineering and Computer Science, 2019</i><br>
            </p>
            <p><span class="smalltext">* Equal Contribution<br><br>Full listing in <a target="_blank" href="https://scholar.google.com/citations?user=MwzQlyIAAAAJ&hl=en">Google Scholar</a>.</span></p>
        </section>
    </body>
</html>
